ui/js/controllers/jobs.js

```diff
@@ -205,6 +205,19 @@ treeherderApp.controller('ResultSetCtrl', [
             });
         };
 
+        $scope.fillAllJobs = function(revision) {
+            if (!window.confirm('This will trigger all jobs for revision ' + revision + '!\n\nAre you sure?')) {
+                return;
+            }
+
+            ThResultSetModel.fillAll($scope.resultset.id, $scope.repoName).catch(function(e) {
+                thNotify.send(
+                    ThModelErrors.format(e, "Failed to fill all jobs"),
```

Comment:
I would keep the 'trigger missing jobs' wording here, e.g.:
"The action 'trigger missing jobs' failed"
or something like that.